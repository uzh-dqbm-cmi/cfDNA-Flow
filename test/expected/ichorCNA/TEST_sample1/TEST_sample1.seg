sample	chr	start	end	event	copy.number	bins	median
TEST_sample1	1	1000001	248000000	NEUT	2	247	-0.000662197023767992
TEST_sample1	2	1000001	242000000	NEUT	2	241	-0.005782321723883
TEST_sample1	3	1000001	198000000	NEUT	2	197	-0.00205728312830769
TEST_sample1	4	2000001	189000000	NEUT	2	187	0.00204285617789869
TEST_sample1	5	1000001	181000000	NEUT	2	180	0.00132583674257692
TEST_sample1	6	1000001	170000000	NEUT	2	169	-0.00499028214643497
TEST_sample1	7	1000001	159000000	NEUT	2	158	0.000432808512266657
TEST_sample1	8	1000001	145000000	NEUT	2	144	0.00332829745933089
TEST_sample1	9	1000001	138000000	NEUT	2	137	0.0040251191640801
TEST_sample1	10	1000001	133000000	NEUT	2	132	-0.000767513761752924
TEST_sample1	11	1000001	135000000	NEUT	2	134	0.00303831575611227
TEST_sample1	12	1000001	132000000	NEUT	2	131	-0.0017716295102117
TEST_sample1	13	20000001	113000000	NEUT	2	93	-3.31819859404293e-05
TEST_sample1	14	20000001	105000000	NEUT	2	85	0.00197504951097693
TEST_sample1	15	25000001	101000000	NEUT	2	76	-0.00217269873157875
TEST_sample1	16	1000001	90000000	NEUT	2	89	0.0175503851724144
TEST_sample1	17	1000001	83000000	NEUT	2	82	-0.00329944355851313
TEST_sample1	18	1000001	80000000	NEUT	2	79	0.00481860143656924
TEST_sample1	19	1000001	58000000	NEUT	2	57	0.0130722597654948
TEST_sample1	20	1000001	64000000	NEUT	2	63	0.0132828932414647
TEST_sample1	21	14000001	46000000	NEUT	2	32	-0.00404098880953001
TEST_sample1	22	19000001	49000000	NEUT	2	30	0.0193292281578303
TEST_sample1	X	4000001	154000000	NEUT	2	150	0.0376442417019156
