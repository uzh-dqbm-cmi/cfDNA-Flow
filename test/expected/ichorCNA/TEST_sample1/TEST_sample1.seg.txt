ID	chrom	start	end	num.mark	seg.median.logR	copy.number	call	subclone.status	logR_Copy_Number	Corrected_Copy_Number	Corrected_Call
TEST_sample1	1	1000001	248000000	247	-0.000662197023767992	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	2	1000001	242000000	241	-0.005782321723883	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	3	1000001	198000000	197	-0.00205728312830769	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	4	2000001	189000000	187	0.00204285617789869	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	5	1000001	181000000	180	0.00132583674257692	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	6	1000001	170000000	169	-0.00499028214643497	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	7	1000001	159000000	158	0.000432808512266657	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	8	1000001	145000000	144	0.00332829745933089	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	9	1000001	138000000	137	0.0040251191640801	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	10	1000001	133000000	132	-0.000767513761752924	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	11	1000001	135000000	134	0.00303831575611227	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	12	1000001	132000000	131	-0.0017716295102117	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	13	20000001	113000000	93	-3.31819859404293e-05	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	14	20000001	105000000	85	0.00197504951097693	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	15	25000001	101000000	76	-0.00217269873157875	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	16	1000001	90000000	89	0.0175503851724144	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	17	1000001	83000000	82	-0.00329944355851313	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	18	1000001	80000000	79	0.00481860143656924	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	19	1000001	58000000	57	0.0130722597654948	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	20	1000001	64000000	63	0.0132828932414647	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	21	14000001	46000000	32	-0.00404098880953001	2	NEUT	FALSE	0.015625	2	NEUT
TEST_sample1	22	19000001	49000000	30	0.0193292281578303	2	NEUT	FALSE	Inf	2	NEUT
TEST_sample1	X	4000001	154000000	150	0.0376442417019156	2	NEUT	FALSE	Inf	2	NEUT
